### PR TITLE
Test content manager

### PR DESCRIPTION
**Pull request by barbaravaldez**
Add test on content manager to create an empty notebook if the content is undefined.
Will add more this week

### CODE REVIEW

File: src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts [hunk 1]
```diff
@@ -131,4 +131,8 @@ suite('Local Content Manager', function (): void {
 		let displayOutput = <nb.IDisplayData>notebook.cells[0].outputs[0];
 		assert.equal(displayOutput.data['text/html'], '<div></div>');
 	});
+	test('Should create a new empty notebook if content is undefined', async function (): Promise<void> {
+		let content = await contentManager.loadFromContentString('');
+		assert.equal(content.cells.length, 0);
```
Comment (corivera) [inline src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts:136]: Do we need to validate if the content itself is undefined? Same for the cells.

File: src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts [hunk 2]
```diff
@@ -131,4 +131,8 @@ suite('Local Content Manager', function (): void {
 		let displayOutput = <nb.IDisplayData>notebook.cells[0].outputs[0];
 		assert.equal(displayOutput.data['text/html'], '<div></div>');
 	});
+	test('Should create a new empty notebook if content is undefined', async function (): Promise<void> {
+		let content = await contentManager.loadFromContentString('');
```
Comment (corivera) [inline src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts:135]: Could you add a case for undefined also?

Comment (barbaravaldez) [reply src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts:135]: Sure!

File: src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts (hunk 1 above)
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts:136]: Since there's only the one assert here it's not strictly necessary, but it'd be good to get in the habit of always including a descriptive message for these so when they fail it's easy to tell what went wrong without needing to look at the code.
